acct-user/subsonic/subsonic-0.ebuild

```diff
@@ -0,0 +1,12 @@
+# Copyright 2019-2020 Gentoo Authors
```

Comment:
Separate commit per package. See comments on other PR too. Do we really need both subsonic and airsonic users + groups?